### PR TITLE
[CU-29qn1qa] docs(mdbook XCVM): update text and summary

### PR DESCRIPTION
**Pull request by Martin-Kiss**
## Issue
https://app.clickup.com/t/29qn1qa

## Description
Replaced old texts with new versions

## Checklist

~~- [ ] I have updated the cargo docs to reflect changes made by this PR _(if applicable)_~~
~~- [ ] I have updated the `book/` to reflect changes made by this PR _(if applicable)_~~

### CODE REVIEW

**Comment by ComposableFiB0t** [comment]
Task linked: [CU-29qn1qa Remove outdated sections from book](https://app.clickup.com/t/29qn1qa)

**Review by 0xBrainjar2 — APPROVED** [review]

File: book/src/products/cross-chain-virtual-machine.md [hunk 1]
```diff
@@ -1,31 +1,40 @@
-# Cross-Chain Virtual Machine
-*The Composable XCVM allows for cross-ecosystem communication.*
+# **[Composable’s Cross-Chain Virtual Machine](https://dali.devnets.composablefinance.ninja/products/cross-chain-virtual-machine.html#cross-chain-virtual-machine)**
```
Comment (cor) [inline book/src/products/cross-chain-virtual-machine.md:1]: why did you make this a link?

File: book/src/products/cross-chain-virtual-machine.md [hunk 2]
```diff
@@ -1,31 +1,40 @@
...
\ No newline at end of file
+During the pre XCVM era, developers who wished to create an interoperable application needed to consider the various chains they wanted their applications to operate between and narrow down their choice to just one as their native chain. Then, they must deploy their project to work with that specific chain’s virtual machine (for instance, the EVM for a project built on Ethereum). An additional burden to developers is finding an interoperability solution that works with Ethereum and the other chains they wish their applications to communicate with, integrate that solution, and use it any time there’s a need to bridge to other applications that are not on Ethereum. Further, suppose they want to deploy that project on additional chains. In that case, DeFi developers must completely recreate their projects and then link them with their existing ones - possibly many times over, depending on the number of instances they desire to create.
+
+
+## **The XCVM and its Vision for Composability**
```
Comment (cor) [inline book/src/products/cross-chain-virtual-machine.md:28]: same as other PR, you should not make `##`'s bold with `**`, as that is done automatically

**Review by cor — CHANGES_REQUESTED** [review]
Same changes reqeusted as #1092

File: book/src/products/cross-chain-virtual-machine.md (hunk 1 above)
Comment (PoisonPhang) [reply book/src/products/cross-chain-virtual-machine.md:1]: This should be addressed, but marketing has been linking titles to their deployed page in the Google doc

Comment (cor) [reply book/src/products/cross-chain-virtual-machine.md:1]: You can can link to titles without making them links in markdown, it's a standard HTML feature. Example: https://dali.devnets.composablefinance.ninja/ecosystem/build-on-composable-ecosystem-development.html#the-applicants-we-are-seeking

Comment (Martin-Kiss) [reply book/src/products/cross-chain-virtual-machine.md:1]: I kept it in based on the edits doc linked in clickup.

Comment (cor) [reply book/src/products/cross-chain-virtual-machine.md:1]: Alright, @Martin-Kiss could you please remove them? :-)

File: book/src/book/src/products/cross-chain-virtual-machine/routing-layer-libraries.md [hunk 3]
```diff
@@ -0,0 +1 @@
+# Routing Layer Libraries
```
Comment (PoisonPhang) [inline book/src/book/src/products/cross-chain-virtual-machine/routing-layer-libraries.md:1]: Can this file be removed?

File: book/src/book/src/products/cross-chain-virtual-machine/writing-smart-contracts-with-cosmwasm.md [hunk 4]
```diff
@@ -0,0 +1 @@
+# Writing Smart Contracts with CosmWasm
```
Comment (PoisonPhang) [inline book/src/book/src/products/cross-chain-virtual-machine/writing-smart-contracts-with-cosmwasm.md:1]: Can this file be removed?

File: book/src/products/cross-chain-virtual-machine.md (hunk 1 above)
Comment (Martin-Kiss) [reply book/src/products/cross-chain-virtual-machine.md]: Will do. 👍

**Review by PoisonPhang — CHANGES_REQUESTED** [review]

File: book/src/book/src/products/cross-chain-virtual-machine/how-the-xcvm-works.md [hunk 5]
```diff
@@ -0,0 +1 @@
+# How the XCVM works
```
[the remark this answers is not in the document]
Comment (Martin-Kiss) [reply book/src/book/src/products/cross-chain-virtual-machine/how-the-xcvm-works.md:1]: Thanks for noticing, I got the file path wrong in SUMMARY.md so the mdbook created dummy files.

**Review by PoisonPhang — DISMISSED** [review]

**Comment by Martin-Kiss** [comment]
bors ping

**Review by PoisonPhang — APPROVED** [review]

**Review by KaiserKarel — APPROVED** [review]

**Review by 0xBrainjar2 — COMMENTED** [review]
the presentation layer diagram is still the old diagram

**Pull request merged**